src/views/router_views/choose_your_package_accounts/choose_your_package_accounts.njk

```diff
@@ -18,4 +19,11 @@
     }) 
   }}
 
+  {{ 
+    govukButton({
+      text: "Confirm and continue",
+      href: "#"
```

Comment:
You don't need the href here just the text. From the gov uk site: 
```
{% from "govuk/components/button/macro.njk" import govukButton %}

{{ govukButton({
  text: "Save and continue"
}) }}
```
https://design-system.service.gov.uk/components/button/#button-second-example-nunjucks